server/src/controller/item.ts

```diff
@@ -53,18 +66,28 @@ export async function createItem(
       await queryRunner.manager.save(itemTag);
     }
 
+    for (let i = 0; i < picturesArray.length; i += 1) {
+      const itemPicture = new ItemPicture();
+      itemPicture.item = item;
+      itemPicture.pictures = picturesArray[i];
+
+      // eslint-disable-next-line no-await-in-loop
+      await queryRunner.manager.save(itemPicture);
+    }
+
     await queryRunner.commitTransaction();
 
     return {
       data: {
         id: item.id,
         supplier: item.supplier ? item.supplier : null,
-        description: item.name,
+        name: item.name,
         price: item.price,
         length: item.length ? item.length : null,
         height: item.height,
         width: item.width,
         tags: tagsArray,
+        pictures: picturesArray,
```

Comment:
Need to return 409 when it is a duplicate item

Reply:
i wasn't able to resolve this.

Reply:
i handled this concern inside the database table.

Reply:
That was handled by the db before, it didn't let having the same name of item per supplier, but it allowed that two different suppliers have the same item name.

Before:
supplier A could have an item called mayonnaise
supplier B could have an item called mayonnaise

With your change:
supplier A could have an item called mayonnaise
supplier B will not be allowed to have an item called mayonnaise

What we where missing, is handle when a supplier tries to create two products with the same name under him, we should return 409 and not 500